docs/howtos/locally-published-components-in-fenix.md

```diff
@@ -8,6 +8,21 @@ from there.
 With support from the upstream project, it's possible to do this in a single step using
 our auto-publishing workflow.
 
+## rust.targets
+
+Both the auto-publishing and manual workflows can be sped up significantly by
+using the `rust.targets` property which limits which architectures the Rust
+code gets build against.  You can set this property by creating/editing the
+`local.properties` file in the repository root and adding a line like
+`rust.targets=x86,linux-x86-64`.  The trick is knowing which targets to put in
+that comma separated list:
+
+  - Use `x86` for running the app on most emulators (in rare cases you'll want `x86_64`)
```

Comment:
what are the rare cases? is it based on the emulator?

Reply:
I *think* so.  I'll update the comment to say that.